internal/topology/check/compatibility.go

```diff
@@ -76,7 +76,8 @@ func ObjectsAreInTheSameNamespace(current, desired client.Object) field.ErrorLis
 	if current.GetNamespace() != desired.GetNamespace() {
 		allErrs = append(allErrs, field.Forbidden(
 			field.NewPath("metadata", "namespace"),
-			fmt.Sprintf("cannot be changed from %v to %v to prevent incompatible changes in %v/%v", current.GetNamespace(), desired.GetNamespace(), current.GetObjectKind().GroupVersionKind().GroupKind().String(), current.GetName()),
+			fmt.Sprintf("metadata.name of %s/%s  cannot be changed from %q to %q because templates must be in the same namespace as the Cluster",
```

Comment:
```suggestion
			fmt.Sprintf("metadata.namespace of %s/%s cannot be changed from %q to %q because templates must be in the same namespace as the Cluster",
```